satellite/repair/repair_test.go

```diff
@@ -63,7 +63,7 @@ func TestDataRepair(t *testing.T) {
 		var (
 			testData         = testrand.Bytes(8 * memory.KiB)
 			minThreshold     = 3
-			successThreshold = 7
+			successThreshold = 6
```

Comment:
why was this updated?

Reply:
the test was failing, this fixed it. asking around for more info.